management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -458,4 +459,27 @@ public void clearSnapshots(@RpcParam(name="snapshotNames") List<String> snapshot
         }
     }
 
+    @Rpc(name = "repair")
+    public void repair(@RpcParam(name="keyspaceName") String keyspace, @RpcParam(name="tables") List<String> tables, @RpcParam(name="full") Boolean full) throws IOException
+    {
+        logger.warn("KEYSPACE = {} ; TABLES = {} ; FULL = {}", keyspace, tables, full);
+        // At least one keyspace is required
+        if (keyspace != null)
+        {
+            // create the repair spec
+            Map<String, String> repairSpec = new HashMap<>();
+
+            // add any specified tables to the repair spec
+            if (tables != null && !tables.isEmpty())
+            {
+                // set the tables/column families
+                repairSpec.put(RepairOption.COLUMNFAMILIES_KEY, String.join(",", tables));
+            }
+
+            // handle incremental vs full
+            repairSpec.put(RepairOption.INCREMENTAL_KEY, Boolean.toString(Boolean.FALSE.equals(full)));
```

Comment:
You'll need to set repair parallelism to `PARALLEL` when incremental repair is being requested, otherwise it'll fail.